stats/cloud/client.go

```diff
@@ -93,28 +93,27 @@ func (c *Client) NewRequest(method, url string, data interface{}) (*http.Request
 }
 
 func (c *Client) Do(req *http.Request, v interface{}) error {
-	var originalBody []byte
-	var err error
-
 	if req.Body != nil {
```

Comment:
Connected to the previous comment, this can be `req.Body != nil && req.GetBody == nil` and it should still work, I think...